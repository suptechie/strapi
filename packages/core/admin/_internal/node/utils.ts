export const prettyTime = (timeInMs: number): string => {
  return Math.ceil(timeInMs) + 'ms';
};
