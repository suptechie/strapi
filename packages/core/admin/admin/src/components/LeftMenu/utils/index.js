// eslint-disable-next-line import/prefer-default-export
export { default as generateModelsLinks } from './generateModelsLinks';
