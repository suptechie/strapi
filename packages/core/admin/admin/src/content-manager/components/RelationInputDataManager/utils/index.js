export { default as connect } from './connect';
export { default as select } from './select';
export { normalizeRelations } from './normalizeRelations';
export { normalizeSearchResults } from './normalizeSearchResults';
