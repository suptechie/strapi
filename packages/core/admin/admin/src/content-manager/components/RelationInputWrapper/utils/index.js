export { default as connect } from './connect';
export { default as select } from './select';
export { normalizeRelations } from './normalizeRelations';
