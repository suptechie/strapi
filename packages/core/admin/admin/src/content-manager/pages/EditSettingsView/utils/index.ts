/* eslint-disable check-file/no-index */
export { createPossibleMainFieldsForModelsAndComponents } from './createPossibleMainFieldsForModelsAndComponents';
export { getInputProps } from './getInputProps';
