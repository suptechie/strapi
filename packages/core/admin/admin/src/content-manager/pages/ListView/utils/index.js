export { default as buildQueryString } from './buildQueryString';
