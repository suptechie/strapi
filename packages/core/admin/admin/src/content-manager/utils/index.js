export { default as arrayMoveItem } from './arrayMoveItem';
export { default as checkIfAttributeIsDisplayable } from './checkIfAttributeIsDisplayable';
export { default as formatLayoutToApi } from './formatLayoutToApi';
export { default as removeKeyInObject } from './removeKeyInObject';
