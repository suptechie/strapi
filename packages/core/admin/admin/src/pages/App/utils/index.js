/* eslint-disable import/prefer-default-export */
export { default as routes } from './routes';
export { default as getUID } from './unique-identifier';
