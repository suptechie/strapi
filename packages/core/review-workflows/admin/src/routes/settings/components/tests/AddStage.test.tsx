import { render } from '@tests/utils';

import { AddStage } from '../AddStage';

describe('AddStage', () => {
  it('should render a list of stages', () => {
    const { container, getByText } = render(<AddStage>Add stage</AddStage>);

    expect(container).toMatchSnapshot();
    expect(getByText('Add stage')).toBeInTheDocument();
  });
});
