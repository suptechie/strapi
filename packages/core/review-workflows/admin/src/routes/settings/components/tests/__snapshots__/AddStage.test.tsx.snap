// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddStage should render a list of stages 1`] = `
.c0 {
  margin-inline-start: -250px;
  position: fixed;
  left: 50%;
  top: 4.6rem;
  z-index: 10;
  width: 50rem;
}

.c2 {
  background: #ffffff;
  padding-block-start: 12px;
  padding-inline-end: 16px;
  padding-block-end: 12px;
  padding-inline-start: 16px;
  box-shadow: 0px 1px 4px rgba(33, 33, 52, 0.1);
}

.c4 {
  font-size: 1.2rem;
  line-height: 1.33;
  font-weight: 600;
  color: currentcolor;
}

.c1 {
  align-items: stretch;
  display: flex;
  flex-direction: column;
  gap: 8px;
}

.c5 {
  align-items: center;
  display: flex;
  flex-direction: row;
  gap: 8px;
}

.c6 {
  border: 0;
  clip: rect(0 0 0 0);
  height: 1px;
  margin: -1px;
  overflow: hidden;
  padding: 0;
  position: absolute;
  width: 1px;
}

.c3 {
  border-radius: 26px;
  color: #8e8ea9;
}

.c3:hover {
  color: #4945ff;
}

.c3:active {
  color: #4945ff;
}

<div>
  <div
    class="c0 c1"
  />
  <button
    border="neutral150"
    class="c2 c3"
  >
    <span
      class="c4"
    >
      <span
        class="c5"
      >
        <svg
          aria-hidden="true"
          fill="currentColor"
          height="2.4rem"
          viewBox="0 0 32 32"
          width="2.4rem"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M16 3a13 13 0 1 0 13 13A13.016 13.016 0 0 0 16 3m5 14h-4v4a1 1 0 0 1-2 0v-4h-4a1 1 0 0 1 0-2h4v-4a1 1 0 0 1 2 0v4h4a1 1 0 0 1 0 2"
          />
        </svg>
        Add stage
      </span>
    </span>
  </button>
  <span
    class="c6"
  >
    <p
      aria-live="polite"
      aria-relevant="all"
      id="live-region-log"
      role="log"
    />
    <p
      aria-live="polite"
      aria-relevant="all"
      id="live-region-status"
      role="status"
    />
    <p
      aria-live="assertive"
      aria-relevant="all"
      id="live-region-alert"
      role="alert"
    />
  </span>
</div>
`;
